src/analysis/FVA/fluxVariability.m

```diff
@@ -133,12 +133,7 @@
 LPproblem.c = model.c;
 LPproblem.lb = model.lb;
 LPproblem.ub = model.ub;
-if ~isfield(model,'csense')
-    LPproblem.csense(1:nMets) = 'E';
-else
-    LPproblem.csense = model.csense;
-end
-
+LPproblem.csense(1:nMets) = 'E';

```

Comment:
These lines have been added in PR #843 (rebase error). May you add them again? Thanks.